app/api/cron/dune/program-calls/route.ts

```diff
@@ -0,0 +1,67 @@
+import { headers } from 'next/headers';
+import { db } from '@/src/db/drizzle';
+import { program_call_stats } from '@/src/db/schema';
+import { NextResponse } from 'next/server';
+import { DuneClient, RunQueryArgs } from '@duneanalytics/client-sdk';
+import { Cluster } from '@utils/cluster';
+import { fetchProgramMetadataIdl, programNameFromIdl } from '@/app/components/instruction/codama/getProgramMetadataIdl';
+import { PROGRAM_INFO_BY_ID } from '@/app/utils/programs';
+
+const { DUNE_API_KEY, DUNE_PROGRAM_CALLS_MV_ID } = process.env;
+
+export async function GET() {
+    const headersList = await headers();
+    if (headersList.get('Authorization') !== `Bearer ${process.env.CRON_SECRET}`) {
+        return NextResponse.json({ error: 'Unauthorized' }, { status: 401 });
+    }
+
+    const client = new DuneClient(DUNE_API_KEY ?? '');
+    const opts: RunQueryArgs = { queryId: Number(DUNE_PROGRAM_CALLS_MV_ID) };
```

Comment:
I'm not sure what the result would be with an empty `MV_ID`. Looks like we should handle this also.